packages/espree/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "espree",
+  "description": "An Esprima-compatible JavaScript parser built on Acorn",
+  "author": "Nicholas C. Zakas <nicholas+npm@nczconsulting.com>",
+  "homepage": "https://github.com/eslint/espree",
+  "main": "dist/espree.cjs",
+  "type": "module",
+  "exports": {
+    ".": [
+      {
+        "import": "./espree.js",
+        "require": "./dist/espree.cjs",
+        "default": "./dist/espree.cjs"
+      },
+      "./dist/espree.cjs"
+    ],
+    "./package.json": "./package.json"
+  },
+  "version": "10.1.0",
+  "files": [
+    "lib",
+    "dist/espree.cjs",
+    "espree.js"
+  ],
+  "engines": {
+    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
+  },
+  "repository": "eslint/espree",
+  "bugs": {
+    "url": "https://github.com/eslint/espree/issues"
+  },
+  "funding": "https://opencollective.com/eslint",
+  "license": "BSD-2-Clause",
+  "dependencies": {
+    "acorn": "^8.12.0",
+    "acorn-jsx": "^5.3.2",
+    "eslint-visitor-keys": "^4.0.0"
+  },
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^17.1.0",
+    "@rollup/plugin-json": "^4.1.0",
+    "@rollup/plugin-node-resolve": "^11.2.0",
+    "c8": "^7.11.0",
+    "chai": "^4.3.6",
+    "eslint": "^9.4.0",
+    "eslint-config-eslint": "^11.0.0",
```

Comment:
```suggestion
```

I think these two can be removed from here as they're not used by this package but from the top level.